tests/testthat/test-apply_theme.R

```diff
@@ -1,12 +1,12 @@
-#' @title Specifications add_theme
+#' @title Specifications apply_theme
 #' @section Last updated by:
 #' Tim Treis
 #' @section Last update date:
 #' 07-MAY-2021
 
 # Specifications ---------------------------------------------------------------
 
-#' T1. The `define_theme` function returns an object that has a valid shape for usage in the `add_theme` function.
+#' T1. The `define_theme` function returns an object that has a valid shape for usage in the `apply_theme` function.
 #' T1.1 No error when no parameters are specified.
 #' T1.2 Not specifying any parameters returns a list.
```

Comment:
an empty list?

Reply:
![image](https://user-images.githubusercontent.com/12818242/120077057-ce1d5680-c0a8-11eb-98d8-c5cea9bc1613.png)
No, some defaults are (silently) automatically filled in.